### PR TITLE
ENH: Add visualization parameters to DSA annotations script

### PR DESCRIPTION
**Pull request by fepegar**
This PR adds the `rescale` and `colormap` parameters to the script in the `girder` module, to improve flexibility and facilitate experimentation.

## Defaults

`Greens` is the default colormap because

1. It goes from white to a dark colour. This is nice because the slides background is typically white as well.
2. Green has the best contrast to the typical tissue colour (~magenta)

Rescaling is not performed by default because it might artificially emphasise attention values that are only slight larger than others. E.g., compare (0.04, 0.06, 0.95) and (0.32, 0.33, 0.34) with their rescaled equivalents: (0, 0.02, 1) and (0, 0.5, 1). The differences between values of the second array are very exaggerated. However, as pointed out in our meetings, this might not be relevant because the values depend on the number of tiles anyway, so we might as well simply use the logits. To be considered in the future.

## Screenshots

Default (no rescaling, `Greens`):

Screenshot 2022-05-31 101513

Rescaling, `Greens:

Screenshot 2022-05-31 101534

No rescaling, `viridis`:

Screenshot 2022-05-31 101555

Rescaling, `viridis`:

Screenshot 2022-05-31 101612

### CODE REVIEW

**Review by vale-salvatelli — APPROVED** [review]

**Comment by fepegar** [comment]
> Thanks @fepegar for the rescaling and visualization examples. Could we also display the 0-1 scale somewhere? This could be useful as different colormaps appear differently, e.g., Greens have lighter color for smaller values, but it is vice-versa for Viridis. One question that we had was which colors are the higher values, perhaps the scale could make this clearer?

I agree a colour bar would definitely be useful. I haven't seen this in DSA, I'm not sure it's supported. But I'll contact the developers and add it in a future PR, if possible.
